lib/src/standardSubstitutions.cc

```diff
@@ -297,37 +297,25 @@ std::string StandardSubstitutions::initVariable(
     return code;
 }
 
-void StandardSubstitutions::currentSourceTimeCondition(
-    std::string &tcCode,
-    const CurrentSource *sc,
-    const DerivedParamNameIterCtx &scmDerivedParams,
-    const ExtraGlobalParamNameIterCtx &scmExtraGlobalParams)
-{
-    substitute(tcCode, "$(t)", "t");
-    value_substitutions(tcCode, sc->getCurrentSourceModel()->getParamNames(), sc->getParams());
-    value_substitutions(tcCode, scmDerivedParams.nameBegin, scmDerivedParams.nameEnd, sc->getDerivedParams());
-    name_substitutions(tcCode, "", scmExtraGlobalParams.nameBegin, scmExtraGlobalParams.nameEnd, sc->getName());
-    checkUnreplacedVariables(tcCode, sc->getName() + " : surrent source timeConditionCode");
-}
-
 void StandardSubstitutions::currentSourceInjection(
     std::string &iCode,
-    const CurrentSource *sc,
-    const VarNameIterCtx &scmVars,
-    const DerivedParamNameIterCtx &scmDerivedParams,
-    const ExtraGlobalParamNameIterCtx &scmExtraGlobalParams,
+    const CurrentSource *cs,
+    const VarNameIterCtx &csmVars,
+    const DerivedParamNameIterCtx &csmDerivedParams,
+    const ExtraGlobalParamNameIterCtx &csmExtraGlobalParams,
     const std::vector<FunctionTemplate> functions,
     const std::string &ftype,
-    const std::string &rng)
+    const std::string &rng,
+    const std::string &popName)
 {
     substitute(iCode, "$(t)", "t");
-    name_substitutions(iCode, "l", scmVars.nameBegin, scmVars.nameEnd, "");
-    value_substitutions(iCode, sc->getCurrentSourceModel()->getParamNames(), sc->getParams());
-    value_substitutions(iCode, scmDerivedParams.nameBegin, scmDerivedParams.nameEnd, sc->getDerivedParams());
-    name_substitutions(iCode, "", scmExtraGlobalParams.nameBegin, scmExtraGlobalParams.nameEnd, sc->getName());
-    substitute(iCode, "$(Iinj)", "Iinj");
+    name_substitutions(iCode, "l", csmVars.nameBegin, csmVars.nameEnd, cs->getName());
```

Comment:
What's this "1" for?

Reply:
I guess it stands for "local"? This "l" is prepended to all neuron vars so I took the same syntax for current sources for the sake of consistency.

Reply:
Duh - of course that's what it's for! Sorry I was being dumb on that one...